### DISCUSSION TITLE
Data Objects Classes

### DISCUSSION THREAD

**Discussion by ghost** [pimcore/pimcore#10027] [Q&A / Support] [answered]
Is there a command line way to save and restore all Data Objects Classes at once instead one by one in GUI mode?

**Comment by NiklasBr** [comment, marked as answer]
As far as a exporting all classed from the CLI I don't know of any method, but it is possible to import with `definition:import:class`.

![image](https://user-images.githubusercontent.com/279826/129329716-e7ac5b89-2bd6-4341-b376-a9f63bdf8b92.png)

You could still try *Bulk Export* and *Bulk Import* in the GUI if that could work for you instead of juggling single files for each class.
